### PR TITLE
enable Pylint

### PR DESCRIPTION
**Pull request by vladak**

### CODE REVIEW

**Comment by vladak** [comment]
will try.

File: dev/install-python-packages.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip3 install --upgrade pip
+pip3 install -r opengrok-tools/src/main/python/requirements.txt
```
Comment (tulinkry) [inline dev/install-python-packages.sh]: this should not be needed, if you install opengrok-tools with pip the requirements are donwloaded automatically

Comment (vladak) [reply dev/install-python-packages.sh]: the opengrok-tools package is not installed during build I think

Comment (tulinkry) [reply dev/install-python-packages.sh]: It is installed when tests phase is on. So why do you need to install its requirements?

Comment (vladak) [reply dev/install-python-packages.sh]: because pylint needs the modules and cannot find them

Comment (tulinkry) [reply dev/install-python-packages.sh]: aha, ok then

Comment (vladak) [reply dev/install-python-packages.sh]: Also, I'd like to be able to check the sources without installing the package. After all, the checks precede the install in the pom file.

**Comment by vladak** [comment]
This will have to be fixed for Windows environment too I believe, will be subject of separate PR.

**Comment by vladak** [comment]
The pylint check for the re-throw action had to be disabled. (https://stackoverflow.com/questions/2228790/why-does-pylint-give-error-e0702-raising-nonetype-on-this-raise-statement)

File: dev/install-python-packages.sh (hunk 1 above)
Comment (vladak) [reply dev/install-python-packages.sh]: Actually, running pylint from venv is reasonable because it does not clash with system installed site packages.

**Pull request merged**